drivers/vulkan/rendering_device_vulkan.cpp

```diff
@@ -8266,7 +8265,7 @@ void RenderingDeviceVulkan::compute_list_dispatch_indirect(ComputeListID p_list,
 	Buffer *buffer = storage_buffer_owner.get_or_null(p_buffer);
 	ERR_FAIL_COND(!buffer);
 
-	ERR_FAIL_COND_MSG(!(buffer->usage & STORAGE_BUFFER_USAGE_DISPATCH_INDIRECT), "Buffer provided was not created to do indirect dispatch.");
+	ERR_FAIL_COND_MSG(!(buffer->usage & VK_BUFFER_USAGE_INDIRECT_BUFFER_BIT), "Buffer provided was not created to do indirect dispatch.");
```

Comment:
Maybe `usage` should be removed, since the VK flags are enough for validation.

Additionally, check `_buffer_allocate()`, which stores `p_usage` in `p_buffer->usage`, but `p_usage` is a set of Vulkan flags, not RD ones.